File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        d={}
        for i in range(len(nums)):
            first=target-nums[i]
            if first in d:
                return [d[first],i]      
            d[nums[i]]=i

File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let stack={}
    for(let i=0;i<nums.length;i++){
        temp=target-nums[i]
        if(temp in stack){
            return [stack[temp],i]
        }
        stack[nums[i]]=i
    }    
};